var/spack/repos/builtin/packages/scr/package.py

```diff
@@ -32,6 +32,7 @@ class Scr(CMakePackage):
     version('develop', branch='develop')
     version('legacy',  branch='legacy')
 
+    version('3.0.0',  sha256='e204d3e99a49efac50b4bedc7ac05f55a05f1a65429500d919900c82490532cc', preferred=True)
     version('3.0rc2', sha256='4b2a718af56b3683e428d25a2269c038e9452db734221d370e3023a491477fad')
     version('3.0rc1', sha256='bd31548a986f050024429d8ee3644eb135f047f98a3d503a40c5bd4a85291308')
```

Comment:
Should we deprecate at least the oldest release candidate (for now) so users get a message if they try to use it? Also so we can eventually remove them and clean up our Spack package.

Reply:
good idea. thank you